src/storage/v2/inmemory/storage.hpp

```diff
@@ -268,16 +267,33 @@ class InMemoryStorage final : public Storage {
 
     void FinalizeTransaction() override;
 
-   private:
+   protected:
     /// @throw std::bad_alloc
-    VertexAccessor CreateVertex(storage::Gid gid);
+    VertexAccessor CreateVertexEx(storage::Gid gid);
 
     /// @throw std::bad_alloc
-    Result<EdgeAccessor> CreateEdge(VertexAccessor *from, VertexAccessor *to, EdgeTypeId edge_type, storage::Gid gid);
+    Result<EdgeAccessor> CreateEdgeEx(VertexAccessor *from, VertexAccessor *to, EdgeTypeId edge_type, storage::Gid gid);
```

Comment:
The way we pulled out the `ReplicationAccessor` caused naming/hiding issue, hence the rename.
These are versions of `CreateX` which are used from the replication code because they can precisely set the `gid` being used (in order to be in line with MAIN).
`Ex` was a temporary name which was inspired by the Win32 API (Ex)tended methods. 
Either we come up with a better name, or try something like the passkey idiom https://accu.org/journals/overload/31/176/mertz/

Reply:
I would opt for a better name but you decide

Reply:
I'll leave a TODO since the API will almost certainly change.